src/components/integrated/index.ts

```diff
@@ -2,3 +2,7 @@ export * from "./OakQuizCheckBox";
 export * from "./OakQuizTextInput";
 export * from "./OakQuizRadioButton";
 export * from "./OakQuizHint";
+export * from "./OakQuizBottomNav";
+export * from "./OakLessonTopNav";
+export * from "./OakQuizFeedback";
+export * from "./OakQuizCounter";
```

Comment:
Every... time... 😆 🤦🏼 

Some sort of lint check might work, or it could be solved with a checklist and good ol' fashioned thoroughness.